dpgen/generator/run.py

```diff
@@ -2469,6 +2467,20 @@ def _make_fp_vasp_configs(iter_index,
         cur_job = json.load(open(os.path.join(modd_path, 'cur_job.json'), 'r'))
         if 'task_min' in cur_job :
             task_min = cur_job['task_min']
+    else:
+        cur_job = {}
+    # support iteration dependent trust levels
+    v_trust_lo = cur_job.get('model_devi_v_trust_lo', jdata.get('model_devi_v_trust_lo', 1e10))
+    v_trust_hi = cur_job.get('model_devi_v_trust_hi', jdata.get('model_devi_v_trust_hi', 1e10))
+    if cur_job.get('model_devi_f_trust_lo'):
+        f_trust_lo = cur_job.get('model_devi_f_trust_lo')
+    else:
+        f_trust_lo = jdata['model_devi_f_trust_lo']
+    if cur_job.get('model_devi_f_trust_hi'):
+        f_trust_hi = cur_job.get('model_devi_f_trust_hi')
+    else:
+        f_trust_hi = jdata['model_devi_f_trust_hi']
```

Comment:
Why not write it as 
```
f_trust_lo = cur_job.get('model_devi_f_trust_lo', jdata['model_devi_f_trust_lo'])
```

Reply:
If written like this, on the condition that trust level only supplied in each `model_devi_job`, it might result in `IndexError` because of the lack of global trust level. It would be confusing for user to set global trust level when local ones set.